x-pack/plugins/security_solution/cypress/e2e/detection_alerts/event_rendered_view.cy.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewRule } from '../../objects/rule';
+import {
+  EVENT_SUMMARY_ALERT_RENDERER_CONTENT,
+  EVENT_SUMMARY_COLUMN,
+  ALERT_RENDERER_HOST_NAME,
+} from '../../screens/alerts';
+import { FIELDS_BROWSER_VIEW_BUTTON } from '../../screens/fields_browser';
+import {
+  DATA_GRID_COLUMN_ORDER_BTN,
+  DATA_GRID_FIELD_SORT_BTN,
+} from '../../screens/common/data_grid';
+import { HOVER_ACTIONS } from '../../screens/timeline';
+import {
+  switchAlertTableToEventRenderedView,
+  waitForAlerts,
+  waitForTopNHistogramToLoad,
+} from '../../tasks/alerts';
+import { createCustomRuleEnabled } from '../../tasks/api_calls/rules';
+import { cleanKibana } from '../../tasks/common';
+import { login, visit } from '../../tasks/login';
+import { ALERTS_URL } from '../../urls/navigation';
+import { TOP_N_ALERT_HISTOGRAM, TOP_N_CONTAINER_CLOSE_BTN } from '../../screens/shared';
+
+describe('Event Rendered View', { testIsolation: false }, () => {
+  before(() => {
+    cleanKibana();
+    login();
+    createCustomRuleEnabled(getNewRule(), 'new custom rule');
+    visit(ALERTS_URL);
+    waitForAlerts();
+    switchAlertTableToEventRenderedView();
+  });
+
+  it('Event Summary Column + Hover Actions', () => {
```

Comment:
This could be broken up into `it('should show the event summary column')` & `it('should render the table hover actions')`. Whether or not the hover actions are actually tested in this file is another discussion as I can see them being tested elsewhere 🤷🏾‍♂️